external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -21,13 +21,17 @@ definition, type on hover). Please see [Troubleshooting](#troubleshooting) for s
 the pre-built move-analyzer binary is not available for your platform or if you want to use move-analyzer
 binary stored in a different location.
 
-If you want to build and test Move code using the extension, you must install the `sui` binary on
+If you want to build, test, and trace Move code using the extension, you must install the `sui` binary on
 your machine - see [here](https://docs.sui.io/guides/developer/getting-started/sui-install) for
 instructions. The extension assumes that the `sui` binary is in your system path, but you can set
 its custom location location using VSCode's settings (`⌘` + `,` on macOS, or use the menu item *Code >
 Preferences > Settings*). Search for the `move.sui.path` user setting, set it to the new location of
 the `sui` binary, and restart VSCode.
 
+In order to trace Move code execution, the `sui` binary must be built with the `tracing` feature flag.
```

Comment:
We have no way of the user checking this outside of looking at the version?

Reply:
One can build a given version with and without the feature flag. The way to check it is to run the tracing command and see if it failed. I added an additional explanation along these lines